cli/examples/gcp.cq.yml

```diff
@@ -5,7 +5,7 @@ spec:
   registry: grpc
   path: "localhost:7777"
   max_goroutines: 10000000
-  tables: ["*"]
+  tables: ["gcp_logging_metrics"]
```

Comment:
was this change intentional?

Reply:
Im going to remove those files (it's just my debug files) (put them in .gitignore)